code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -186,67 +266,117 @@
 	desc = "An experimental suit of armor that violently throws back attackers."
 	reactivearmor_cooldown_duration = 5 SECONDS
 	var/repulse_force = MOVE_FORCE_EXTREMELY_STRONG
+	cooldown_message = "<span class='danger'>The repulse generator is still recharging! It fails to generate a strong enough wave!</span>"
+	emp_message = "<span class='warning'>The repulse generator is reset to default settings...</span>"
```

Comment:
Can you use the ``span_[class]()`` instead, you use these scattered across and it would be better if we moved away from it.